lazylibrary/src/main/java/org/wisdomrider/lazylibrary/LazyBase.kt

```diff
@@ -48,52 +47,40 @@ open class LazyBase : AppCompatActivity() {
     }
 
     fun <T> Call<T>.fetch(
+        context: Context,
         response: (response: T?) -> Unit,
         failure: ((t: Throwable) -> Unit)? = null,
         showDialog: Boolean = true
     ) {
-        var dialog = DataDialog(this@LazyBase)
+        var dialog = DataDialog(context)
```

Comment:
pass activity context here not context

Reply:
I think I have remove DataDialog so you need to review Again ?